curvefs/src/client/common/config.h

```diff
@@ -80,6 +80,11 @@ struct SpaceAllocServerOption {
     uint64_t rpcTimeoutMs;
 };
 
+struct KVClientManagerOpt {
+    int setThreadPooln = 4;
+    int getThreadPooln = 4;
```

Comment:
getThreadPooln where does this param use?

Reply:
Here is a reserved parameter, libmemcached, the current overall reading process is synchronous. Also waiting for pr #2080